### PR TITLE
Add mrc recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
@tlambert03 do you agree with being added as a maintainer?

**Comment by tlambert03** [comment]
Yep thanks

File: recipes/mrc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+  home: https://github.com/tlambert03/mrc
+  summary: Read and write .mrc and .dv (deltavision) image file format
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/napari
```
Comment (jaimergp) [inline recipes/mrc/meta.yaml:43]: This one does not depend on napari, but I assume is a dependency of a plugin. Do we still want the team to be maintaining this one too?

Comment (tlambert03) [reply recipes/mrc/meta.yaml:43]: No not On this one. That would just be napari-dv

Comment (tlambert03) [reply recipes/mrc/meta.yaml:43]: Just me here

**Pull request merged**